department-of-veterans-affairs/caseflow
# Add feature toggle and placeholder for reentry form flow

Because adding the RAMP re-entry form is going to be a big
feature change implemented over many PRs and revisions,
were gonna slide it behind a feature flag.

## Test instructions
- go to the intake start page, nothing should have changed
- enable the `:intake_reentry_form` feature toggle
- go to the intake start page and see the form select page (it wont do anything for now)

connects #3956
